build.gradle

```diff
@@ -70,6 +69,8 @@ dependencies {
 
     // integration tests
     implementation project(':iexec-blockchain-adapter-api-library')
+
+    testImplementation 'org.springframework.security:spring-security-test'
```

Comment:
With the jvm test suite plugin
https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html, we avoid the `testImplementation` dependencies here and move them as `implementation` dependencies to the testing block.